settings.py

```diff
@@ -551,6 +555,7 @@ def JINJA_CONFIG():
 # in DOMAIN_METHODS.
 # http://github.com/jbalogh/zamboni/blob/d4c64239c24aa2f1e91276909823d1d1b290f0ee/settings.py#L254
 STANDALONE_DOMAINS = ['django', 'javascript']
+STATICI18N_ROOT = 'build/locale'
 STATICI18N_DOMAIN = 'javascript'
```

Comment:
It's more than Django's javascript, it's l10n strings for all of MDN's javascript also. I find this name suitable.